node-backend/app/routes/badgesRoutes_v1.js

```diff
@@ -1,26 +1,47 @@
 const express = require("express");
 const bodyParser = require("body-parser");
 const BadgesController = require("../controllers/v1/badgesController");
+const jwtValidate = require("../jwtTokenValidation/jwtValidation");
 const badgesRouter = express.Router();
 badgesRouter.use(bodyParser.urlencoded({ extended: true }));
-//for create core value
+
+async function authorizedRole(req, res, next) {
+  const authHeader = req.headers["authorization"];
```

Comment:
pass req.headers["authorization"] to jwtValidate.getData so we will not need to do this splitting thing everywhere in code